vocabularies/Common.xml

```diff
@@ -122,10 +122,15 @@
         <Annotation Term="Core.Description" String="The point in time represented by the annotated property or parameter shall be interpreted in the given time zone" />
         <Annotation Term="Core.LongDescription">
           <String>Time zones shall be specified according to the [IANA](https://www.iana.org/time-zones) standard.
-            If this annotation is absent or null or an empty string, points in time are typically interpreted in the current user's or default time zone.</String>
+            If this annotation is absent or null or an empty string, points in time are typically interpreted in the current user's or default time zone.
+            The annotation value can be a path expression resolving to a property tagged as [`IsTimezone`](#IsTimezone).</String>
```

Comment:
How has this to be interpreted? Is isTimezone mandatory for a property referenced in Timezone? And would the UI need to check also for the presence of isTimezone? Somehow that seems like overkill. How about "... resolving to a property that may be tagged as ..."

Reply:
Similar `Measures.Unit` and `Common.IsUnit`.